xen.spec

```diff
@@ -405,7 +426,11 @@ ln -s ../lib/%{name}/bin/qemu-io  %{buildroot}/usr/bin/
 ln -s ../lib/%{name}/bin/qemu-nbd %{buildroot}/usr/bin/
 
 # "build" combined OVMF image based on system package
+%if 0%{?rhel} >=7
+cat /usr/share/OVMF/OVMF_{VARS,CODE.secboot}.fd > %{buildroot}/usr/lib/xen/boot/ovmf.bin
```

Comment:
Are you sure about `secboot` part? According to [package OVMF-20150414-2.gitc9e5618.el7.noarch.rpm content](http://cbs.centos.org/koji/rpminfo?rpmID=33246), there is no such file. Also, if that version enforce "Secure Boot", it would break not signed (by Microsoft) operating systems.
On the other hand, this file is used only in dom0, so it doesn't really matter in VM package.

Reply:
Yes. Have a look on the latest version : http://mirror.centos.org/centos/7/os/x86_64/Packages/OVMF-20170228-5.gitc325e41585e3.el7.noarch.rpm

I checked twice because, indeed, it was misleading...by the way, it seems that this is the only version provided for CentOS. There is the repo https://www.kraxel.org/repos/ which provides the latest package but, as you pointed out, it doesn't really matter in a VM package.